src/autoschedulers/adams2019/AutoSchedule.cpp

```diff
@@ -156,28 +115,32 @@ struct ProgressBar {
     const bool draw_progress_bar = isatty(2) && aslog::aslog_level() >= ProgressBarLogLevel;
 };
 
-// Get the HL_RANDOM_DROPOUT environment variable. Purpose of this is described above.
-uint32_t get_dropout_threshold() {
-    string random_dropout_str = get_env_variable("HL_RANDOM_DROPOUT");
-    if (!random_dropout_str.empty()) {
-        return atoi(random_dropout_str.c_str());
-    } else {
-        return 100;
+#ifdef HALIDE_ALLOW_LEGACY_AUTOSCHEDULER_API
```

Comment:
This is a bit hidden. In the environment variables list about, we should keep the descriptions, and say that if you want to use the following environment variables to control more features of the autoscheduler, you should #define HALIDE_ALLOW_LEGACY_AUTOSCHEDULER_API here, but this is deprecated and better done using the parameters struct

Reply:
Good point, will do.

Reply:
Done, PTAL